src/test/kotlin/com/ampnet/walletservice/controller/ControllerTestBase.kt

```diff
@@ -48,6 +48,8 @@ import org.springframework.web.context.WebApplicationContext
 import java.time.ZonedDateTime
 import java.util.UUID
 
+const val coop = "ampnet"
```

Comment:
Set as `protected val` inside ControllerTestBase

Reply:
if you use `const val` see: https://kotlinlang.org/docs/reference/coding-conventions.html#property-names